EventsExpress/ClientApp/src/containers/users.js

```diff
@@ -8,16 +8,20 @@ import UsersFilterWrapper from '../containers/user-filter';
 class UsersWrapper extends Component {
     componentDidMount() {
         this.getUsers(this.props.location.search);
+        this.getCount();
+        this.intervalId = setInterval(this.getCount, 120000);
```

Comment:
Why is this interval required?

Reply:
Interval is used because we need to somehow synchronize information between the database and the user's page. Therefore I have created this kind of logic to implement counting in real-time.